docs/content/docs/troubleshooting/osm_config_map.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "OSM ConfigMap Troubleshooting"
+description: "OSM Config Map Troubleshooting Guide"
+type: docs
+---
+
+# OSM ConfigMap Troubleshooting Guide
+
+## OSM Mesh Upgrade Timing Out
+### Insufficient CPU
+If the `osm mesh upgrade` command is timing out, it could be due to insufficient CPU.
```

Comment:
Why would updating the config-map result in insufficient CPUs? 

It seems like this is more to do with upgrades of the controller. Doesn't seem like this should be in the troubleshooting doc for the config-map.